frontend/src/components/homepage/PreviewModal.tsx

```diff
@@ -67,12 +66,40 @@ const PreviewModal = ({
     },
   });
 
+  const storyContents = content.map((c: string, index: number) => (
+    <Box display="flex" padding="8px 8px 8px 8px">
+      <Box
+        w="3%"
+        text-align="center"
+        float="left"
+        margin="0px 10px 0px 0px"
+        color="black"
+      >
+        <Text fontSize="12px" as="b" align="center">
+          {index + 1}
+        </Text>
+      </Box>
+      <Box
+        bg="blue.50"
+        w="97%"
```

Comment:
this seems a little weird, since it's only 3% from 100%. is 100% outrageous?

Reply:
nit: err on the side of using long-form attribute names with chakra ui - imo it's more readable. ie change `bg` and `w`